packages/examples/src/registry.json

```diff
@@ -530,5 +530,8 @@
   "alloy-models/generic": {
     "name": "Alloy: Generic Style",
     "gallery": false
-  }
+  },
+  "Dynamics/Lyapunov": {
+    "name" : "Lyapunov Exponent",
+    "gallery": true
```

Comment:
```suggestion
    "gallery": true
  }  
```

Reply:
oops! Thought I'd caught all the dumb mistakes with the first fix, thanks for checking it over!